src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-WebApplication/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Lets change the defaults in config instead of code.

```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Warning",
      "Microsoft.Hosting.Lifetime": "Information"
    }
  },
  "AllowedHosts": "*",
  "Kestrel": {
    "EndpointDefaults": {
      "Protocols": "Http2"
    }
  }
}
```